src/test_workflow/integ_test/integ_test_suite_opensearch_dashboards.py

```diff
@@ -81,6 +97,38 @@ def __setup_cluster_and_execute_test_config(self, config: str) -> int:
             os.chdir(self.work_dir)
             return self.execute_integtest_sh(endpoint, port, security, config)
 
+    def multi_execute_integtest_sh(self, cluster_endpoints: list, security: bool, test_config: str) -> int:
+        script = ScriptFinder.find_integ_test_script(self.component.name, self.repo.working_directory)
+
+        def custom_node_endpoint_encoder(node_endpoint: NodeEndpoint) -> dict:
+            return {"endpoint": node_endpoint.endpoint, "port": node_endpoint.port, "transport": node_endpoint.transport}
+        if os.path.exists(script):
+            single_node = cluster_endpoints[0].data_nodes[0]
+            cmd = f"bash {script} -b {single_node.endpoint} -p {single_node.port} -s {str(security).lower()} -t {self.component.name} -v {self.bundle_manifest.build.version}"
```

Comment:
Hi @tianleh , seems there is an issue in running integtests when -t is passed from python code base @line 107.

Reply:
Checked with @peterzhuamazon . Please raise the PR to FTRep once ready.